dom.bs

```diff
@@ -5668,6 +5756,34 @@ method steps are to return the result of running the
 <a>internal <code>createElementNS</code> steps</a>, given <a>this</a>, <var>namespace</var>,
 <var>qualifiedName</var>, and <var>options</var>.
 
+<p>To <dfn>flatten element creation options</dfn>, given a string or {{ElementCreationOptions}}
+dictionary <var>options</var> and a <a for=/>document</a> <var>document</var>:
+
+<ol>
+ <li><p>Let <var>registry</var> be null.
+
+ <li><p>Let <var>is</var> be null.
+
+ <li>
+  <p>If <var>options</var> is a dictionary:
```

Comment:
I think we're missing the alternate case here, for when `options` is a string - `is` is never set to `options` AFAICT.

Reply:
That is correct? It never was either.